tests/DevApps/ContosoWorker/Program.cs

```diff
@@ -0,0 +1,19 @@
+using ContosoWorker;
+using Microsoft.Identity.Abstractions;
+using Microsoft.Identity.Web;
+using Microsoft.Identity.Web.TokenCacheProviders.InMemory;
+
+var builder = Host.CreateApplicationBuilder(args);
+
+builder.Services.AddHostedService<Worker>();
+
+builder.Services.AddTokenAcquisition(isTokenAcquisitionSingleton:true)
```

Comment:
We could introduce a:
TokenAcquirerFactory.GetDefault(hostBuilder) adds the token acquisition, the cache and sets the MicrosoftIdentityOptions.

Note that the AddTokenAcquisition in this case uses an Sdk host, which means it's not drawing ASP.NET Core services.